requirements/base.in

```diff
@@ -4,6 +4,6 @@ Django>=1.8,<2.0          # Web application framework
 django-model-utils        # Provides TimeStampedModel abstract base class
 djangorestframework>=3.2.0,<3.7.0  # REST API framework
 edx-opaque-keys[django]   # Create and introspect course and xblock identities
-edx-drf-extensions>=2.0.0,<3.0.0 # Provides JWT authentication
+edx-drf-extensions>=1.11.0,<3.0.0 # Provides JWT authentication
```

Comment:
This file is pulled directly into setup.py, so this is required to allow edx-solutions/edx-platform to install the older version.